src/app/sidebar/sidebar.component.scss

```diff
@@ -81,74 +78,116 @@ a[data-toggle="collapse"] {
   right: 10px;
   transform: translateY(-50%);
 }
-.bgColor{
+
+.bgColor {
   background-image: url('/assets/images/company-logos/sideimg.svg');
   background-position: bottom;
   background-repeat: no-repeat;
   background-size: contain;
   width: 100%;
 }
+
 #sidebar {
   border: none;
-  li:active{
-  color: #64c2e1;
-}
-li:focus{
-  outline:#64c2e1  ;
-  border-radius: 20px;
-  color: #00ABFF;
-  
-}
-}
-.dharma:hover{
-.fa-dharmachakra{
-  -webkit-animation:spin 4s linear infinite;
-  -moz-animation:spin 4s linear infinite;
-  animation:spin 4s linear infinite;
+
+  li:active {
+    color: #64c2e1;
+  }
+
+  li:focus {
+    outline: #64c2e1;
+    border-radius: 20px;
+    color: #00ABFF;
+  }
 }
-@-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
-@-webkit-keyframes spin { 100% { -webkit-transform: rotate(360deg); } }
-@keyframes spin { 100% { -webkit-transform: rotate(360deg); transform:rotate(360deg); } }
+
+.dharma:hover {
+  .fa-dharmachakra {
+    transform: rotate(120deg);
+    transition: 1.5s;
+  }
 }
-.sidebarBottom{
-  color:#004ED6;
+
+.sidebarBottom {
+  color: #004ED6;
   font-size: 14px;
   position: absolute;
-  bottom: 0; 
+  bottom: 0;
 }
-.ciCdAnime{
-  box-sizing: border-box;
-  .line1{
-    background: #212428; padding-top:8px; border-radius: 2px; transition: 700ms; padding-left: 2px;padding-right: 2px;
+
+.ciCdAnime {
+  .line1 {
+    background: #212428;
+    padding-top: 8px;
+    border-radius: 2px;
+    transition: 700ms;
+    padding-left: 2px;
+    padding-right: 2px;
   }
-  .line2{
-    background: #212428; padding-top:1px;border-radius: 2px; transition: 700ms; padding-left: 2px;padding-right: 2px;
+
+  .line2 {
+    background: #212428;
+    padding-top: 1px;
+    border-radius: 2px;
+    transition: 700ms;
+    padding-left: 2px;
+    padding-right: 2px;
   }
-  .line3{
-    background: #212428; padding-top:17px;border-radius: 2px; transition: 700ms; padding-left: 2px;padding-right: 2px;
+
+  .line3 {
+    background: #212428;
+    padding-top: 17px;
+    border-radius: 2px;
+    transition: 700ms;
+    padding-left: 2px;
+    padding-right: 2px;
   }
 }
-.ciCd:hover{
-  .line1{
-     padding-top:15px; transition: 500ms;
+
+.ciCd:hover {
+  .line1 {
+    padding-top: 15px;
+    transition: 500ms;
   }
-  .line2{
-     padding-top:10px; transition: 500ms;
+
+  .line2 {
+    padding-top: 10px;
+    transition: 500ms;
   }
-  .line3{
-     padding-top:5px; transition: 500ms;
+
+  .line3 {
+    padding-top: 5px;
+    transition: 500ms;
   }
 }
-.ciCd:active{
-  .line1{
-     padding-top:5px; transition: 50ms;
-  }
-  .line2{
-     padding-top:15px; transition: 50ms;
-  }
-  .line3{
-     padding-top:8px; transition: 50ms;
+
+.active-link {
```

Comment:
why this is needed if you have this `ciCd:active`?

Reply:
here i have some bars , for changing its background color i use like this

Reply:
change the class name accordingly